storage/innobase/xtrabackup/test/t/bug1630841.sh

```diff
@@ -98,6 +102,10 @@ start_server
 mysql -e "CREATE USER pbx@'localhost' IDENTIFIED BY 'password1'"
 mysql -e "GRANT BACKUP_ADMIN, PROCESS, RELOAD, LOCK TABLES, REPLICATION CLIENT ON *.* TO pbx@'localhost'"
 mysql -e "GRANT SELECT ON performance_schema.log_status TO pbx@'localhost'"
+if is_server_version_higher_than 8.0.23
```

Comment:
here too, 8024?

Reply:
@satya-bodapati  the higher than doesn't include equal :D got tricked on that one too - https://github.com/percona/percona-xtrabackup/blob/percona-xtrabackup-8.0.23-16/storage/innobase/xtrabackup/test/inc/common.sh#L783

Reply:
brain freeze moment :flushed: